packages/app/src/app/pages/Pro/Upgrade.tsx

```diff
@@ -162,6 +173,24 @@ export const ProUpgrade = () => {
       team.subscription?.paymentProvider === SubscriptionPaymentProvider.Paddle
   );
 
+  const getPricePerMonth = (
+    type: 'individual' | 'team',
+    period: 'year' | 'month'
+  ) => {
+    const priceInCurrency =
+      pro?.prices?.[type]?.[period]?.[currency.toLowerCase()] ||
+      pro?.prices?.[type]?.[period]?.usd; // fallback to usd
+
+    // Divide by 12 if the period is year to get monthly price for yearly
+    // subscriptions
+    const price = period === 'year' ? priceInCurrency / 12 : priceInCurrency;
```

Comment:
I think this can resolve to `NaN` or `undefined` if we don't have the data that we expect (or it's loading and the connection is slow) and because `formatCurrency` only checks if the `amount` is `null`, it results in the following in a slow connection:

![Screen Shot 2023-03-28 at 11 36 05](https://user-images.githubusercontent.com/24959348/228273810-195f90a8-e586-4aa0-a411-9aaf1bcdc308.png)

[This](https://www.speedtest.net/global-index/india) shows that India ranks 81st in internet speed, so we could add a loading state, not render anything or show a fallback to avoid having this issue.

Reply:
This was actually a bug when loading the active `subscription`! I've fixed this in [f28e0ce](https://github.com/codesandbox/codesandbox-client/pull/7543/commits/f28e0cebc0dec00aa014abcd1999b63b03751a49), among other things.